third_party/sorbet_test_corpus_runner.patch

```diff
@@ -1,13 +0,0 @@
-diff --git a/test/BUILD b/test/BUILD
```

Comment:
Q: This patch file isn't needed anymore because we can just run the test files directly?

Reply:
Yep, no need to patch them to use the plugin injector now.